### PR TITLE
Add edgetest plugins

### PR DESCRIPTION
**Pull request by fdosani**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by fdosani** [comment]
@ak-gupta would you mind providing your approval to be a maintainer?

**Comment by fdosani** [comment]
@conda-forge/help-python ready for review!

File: recipes/edgetest-conda/meta.yaml
Comment (dopplershift) [inline recipes/edgetest-conda/meta.yaml:24]: It doesn't look like this is actually needed at runtime? You could move it to `host` if you like since it is used by `setup.py`.

File: recipes/edgetest-hub/meta.yaml
Comment (dopplershift) [inline recipes/edgetest-hub/meta.yaml:24]: Here too.

File: recipes/edgetest-pip-tools/meta.yaml
Comment (dopplershift) [inline recipes/edgetest-pip-tools/meta.yaml:24]: And here.

**Review by dopplershift — COMMENTED** [review]
It doesn't seem like setuptools *should* be needed at runtime, but you definitely added them to fix the builds. Unfortunately, I can't seem to get to the logs for those failures--can you describe what the problem was?

Also need confirmation from @ak-gupta that they are willing to be listed as a maintainer for these recipes.

**Comment by ak-gupta** [comment]
> @ak-gupta would you mind providing your approval to be a maintainer?

Yep, I approve being a maintainer

**Comment by fdosani** [comment]
@dopplershift would you prefer it if we fixed the https://github.com/conda-forge/cerberus-feedstock/blob/master/recipe/meta.yaml#L26 first which would allow us to remove setuptools? I think it would be better to fix the upstream so happy to hold off until I can get access to be a maintainer.

File: recipes/edgetest-hub/meta.yaml
Comment (fdosani) [reply recipes/edgetest-hub/meta.yaml]: Removed from run

File: recipes/edgetest-pip-tools/meta.yaml
Comment (fdosani) [reply recipes/edgetest-pip-tools/meta.yaml]: Removed from run

**Comment by fdosani** [comment]
latest version of edgetest hasn't synced yet: https://anaconda.org/conda-forge/edgetest/files (v2021.12.3)
Will restart CI once it is up.

**Comment by fdosani** [comment]
@conda-forge-admin, please restart ci

**Comment by fdosani** [comment]
@dopplershift I think we are ready for your review. Thank you again!

**Comment by fdosani** [comment]
Friendly ping to @conda-forge/help-python. Ready for review!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**
